x/ccv/utils/utils.go

```diff
@@ -41,30 +42,37 @@ func GetChangePubKeyAddress(change abci.ValidatorUpdate) (addr []byte) {
 	return pk.Address()
 }
 
-// SendIBCPacket sends an IBC packet with packetData
-// over the source channelID and portID
-func SendIBCPacket(
+// PrepareIBCPacketSend prepares the send an IBC packet with packetData
+// over the source channelID and portID. If successful, it returns the
+// packet and the channel capability.
+func PrepareIBCPacketSend(
```

Comment:
Let's put combine and prepare back into one step

Reply:
Done.